bedrock/sitemaps/utils.py

```diff
@@ -150,10 +157,35 @@ def get_static_urls():
     return urls
 
 
+def _get_vrc_urls():
+
+    urls = defaultdict(list)
+
+    for entry in ContentfulEntry.objects.filter(
+        content_type=CONTENT_TYPE_PAGE_RESOURCE_CENTER,
+        classification=CONTENT_CLASSIFICATION_VPN,
+    ):
+        _path = f"{VRC_ROOT_PATH}{entry.slug}"
+        urls[_path].append(entry.locale)  # In the future one, slug may support multiple locales
+
+    if urls:
+        # We only show the VRC listing page if there are VRC articles in the DB
+        urls[VRC_ROOT_PATH] = ["en-US"]  # TODO: support more locales
```

Comment:
The VRC index page is already listed in the sitemap actually, because it has a static entry in a urls.py file. See https://github.com/mozmeao/www-sitemap-generator/blob/master/data/sitemap.json#L9082-L9084

Reply:
Ah yeah, makes sense - will pull it from my code